CMakeLists.txt

```diff
@@ -8,16 +8,10 @@ project(open62541 C)
 
 # Find Python and Python-lxml
 find_package(PythonInterp REQUIRED)
-execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
```

Comment:
Jep, because the get_python_lib command only returns the global path, i.e. if you install packages as root.
Since the new travis containers don't allow sudo installs, we have to use `pip install --user` and thus the lxml package isn't by this snippet.
That's why I changed it to `import lxml` and check the return value. Then you are 100% sure that python can find lxml or can't find it.